src/chip/checkbox-chip-group.svelte

```diff
@@ -0,0 +1,51 @@
+<script>
+  import s from '../_utils/plural-s.js';
+  import CheckboxChip from './checkbox-chip.svelte';
+
+  let _class = null;
+  export { _class as class };
+  export let checkboxClass = null;
+  export let labelClass = null;
+
+  export let items;
+  export let name;
+  export let max = null;
+
+  $: currentChecked = items.reduce((acc, elt) => acc + elt.checked, 0);
+  const maxReachedTooltip = { title: `Can only select ${max} value${s(max)}.` };
+
+  if (!items || items.length === 0) {
+    console.error('Must have at least one item in the checkbox chip group');
+  }
+</script>
+
+{#if items != null && items.length !== 0}
+  <div class={_class} role="group">
+    {#each items as item (item.value)}
+      <CheckboxChip
+        {name}
+        value={item.value}
+        bind:checked={item.checked}
+        disabled={item.disabled || (!item.checked && max != null && currentChecked >= max)}
+        class={checkboxClass}
+        on:change
+        {...$$restProps}
+        {...(
+          !item.disabled
+          && !item.checked
+          && max != null
+          && currentChecked >= max
+          ? maxReachedTooltip : {}
+        )}
```

Comment:
How about moving this whole condition into an appropriately-named reactive variable and using the `title` prop directly instead of the weird spread?
Also, doesn't having it after the `$$restProps` spread make it override it? In that case, it would be better to keep it before it so the user can override the tooltip manually (until better i18n support and so on)